pages/file-dropzone/container.page.tsx

```diff
@@ -0,0 +1,120 @@
+    setFiles(newValue);
+
+    setSelectedItems([]);
+  };
+
+  const { areFilesDragging } = useFilesDragging();
+
+  return (
+    <Box margin="xl">
+      <SpaceBetween size="xl">
+        <Header variant="h1">File dropzone: in container</Header>
+        <Checkbox checked={onlyVisibleOnDrag} onChange={() => setUrlParams({ onlyVisibleOnDrag: !onlyVisibleOnDrag })}>
+          Only visible on file drag
+        </Checkbox>
+        <Container
+          header={
+            <Header
+              actions={
+                <Button disabled={selectedItems.length === 0} onClick={removeFiles}>
+                  {selectedItems.length > 1 ? 'Remove files' : 'Remove file'}
+                </Button>
+              }
+            >
+              Attachments
+            </Header>
+          }
+        >
+          <SpaceBetween size="l">
+            {/* {!onlyVisibleOnDrag && (
+              <FileDropzone onChange={(event: any) => handleFilesChange(event.detail.value)}>
+                <Box fontWeight="bold" color="inherit">
+                  Drop files here
+                </Box>
+              </FileDropzone>
+            )} */}
+            {areFilesDragging && onlyVisibleOnDrag ? (
+              <FileDropzone onChange={(event: any) => handleFilesChange(event.detail.value)}>
+                <Box fontWeight="bold" color="inherit">
+                  Drop files here
+                </Box>
+              </FileDropzone>
+            ) : (
+              <Table
+                variant="embedded"
+                empty={'No files uploaded'}
+                selectionType={'multi'}
+                onSelectionChange={({ detail }) => setSelectedItems(detail.selectedItems)}
+                selectedItems={selectedItems}
+                trackBy="name"
+                columnDefinitions={[
+                  {
+                    id: 'file-name',
+                    header: 'File name',
+                    cell: item => item.name || '-',
+                    sortingField: 'name',
+                    isRowHeader: true,
+                  },
+                  {
+                    id: 'file-type',
+                    header: 'File type',
+                    cell: item => item.type || '-',
+                    sortingField: 'alt',
+                  },
+                  {
+                    id: 'file-size',
+                    header: 'File size',
+                    cell: item => item.size || '-',
+                  },
+                ]}
+                enableKeyboardNavigation={true}
+                ariaLabels={{
+                  selectionGroupLabel: 'group label',
+                  allItemsSelectionLabel: ({ selectedItems }) => `${selectedItems.length} item selected`,
+                  itemSelectionLabel: ({ selectedItems }, item) =>
+                    `${item.name} is ${selectedItems.indexOf(item) < 0 ? 'not ' : ''}selected`,
+                }}
+                items={files.map(file => ({
```

Comment:
Why this mapping is needed? It will be the same if the files array is passed as is.